dashboard/app/helpers/levels_helper.rb

```diff
@@ -315,8 +315,11 @@ def app_options
       end
     end
 
+    # TODO: Ues correct app for a lesson with music and standalone_video levels.
     @app_options =
-      if @level.is_a? Blockly
+      if @script_level&.lesson&.use_lab_container
+        {app: 'lablab'}
```

Comment:
are we removing some app options by doing this? I believe the only one we are relying on is channel, but we want to make sure we are still sending it.

Reply:
Oh, do we still need channel ID rendered into the page?

Reply:
Yeah, we will for standalone projects

Reply:
Got it.  I think I've now got that working.